src/tests/cxxfrontend.cc

```diff
@@ -562,6 +562,1124 @@ void test_optional()
+                    (void)ad->anonalignsize;
+                    continue;
+                }
+                if (AttribDeclaration *attrib = sym->isAttribDeclaration())
+                {
+                    (void)attrib->include(NULL);
+                    continue;
+                }
+                if (sym->isTemplateMixin() || sym->isNspace())
+                {
+                    if (ScopeDsymbol *scopesym = sym->isScopeDsymbol())
+                    {
+                        (void)scopesym->members;
+                        continue;
+                    }
+                }
+            }
+        }
+        (void)t->sym->storage_class;
+        t->sym->type->accept(this);
+        for (size_t i = 0; i < t->sym->vtbl.length; i++)
+            t->sym->vtbl[i]->isFuncDeclaration()->accept(this);
+        for (size_t i = 0; i < t->sym->baseclasses->length; i++)
+        {
+            BaseClass *bc = (*t->sym->baseclasses)[i];
+            bc->sym->accept(this);
+        }
+        visitUserAttributes(t->sym);
+    }
+    void visit(Statement *) { assert(0); }
+    void visit(ScopeGuardStatement *) { }
+    void visit(IfStatement *s)
+    {
+        s->condition->accept(this);
+        s->condition->type->accept(this);
+        if (s->ifbody)
+            s->ifbody->accept(this);
+        if (s->elsebody)
+            s->elsebody->accept(this);
+    }
+    void visit(PragmaStatement *) { }
+    void visit(WhileStatement *) { assert(0); }
+    void visit(DoStatement *s)
+    {
+        s->getRelatedLabeled()->accept(this);
+        if (s->_body)
+            s->_body->accept(this);
+        s->condition->accept(this);
+        s->condition->type->accept(this);
+    }
+    void visit(ForStatement *s)
+    {
+        s->getRelatedLabeled()->accept(this);
+        if (s->_init)
+            s->_init->accept(this);
+        if (s->condition)
+        {
+            s->condition->accept(this);
+            s->condition->type->accept(this);
+        }
+        if (s->_body)
+            s->_body->accept(this);
+        if (s->increment)
+            s->increment->accept(this);
+    }
+    void visit(ForeachStatement *) { assert(0); }
+    void visit(ForeachRangeStatement *) { assert(0); }
+    void visit(BreakStatement *s)
+    {
+        if (s->ident)
+        {
+            LabelDsymbol *sym = func->searchLabel(s->ident, s->loc);
+            LabelStatement *label = sym->statement;
+            label->statement->getRelatedLabeled()->accept(this);
+        }
+    }
+    void visit(ContinueStatement *s)
+    {
+        if (s->ident)
+        {
+            LabelDsymbol *sym = func->searchLabel(s->ident, s->loc);
+            LabelStatement *label = sym->statement;
+            label->statement->accept(this);
+        }
+    }
+    void visit(GotoStatement *s)
+    {
+        assert(s->label->statement != NULL);
+        assert(s->tf == s->label->statement->tf);
+        (void)s->label->ident;
+    }
+    void visir(LabelStatement *s)
```

Comment:
Hi I was randomly browsing and `visir` is probably wrong!

Reply:
Good thing the test it isn't executed then. :-)